.github/workflows/backend-testing.yml

```diff
@@ -22,10 +22,11 @@ jobs:
     steps:
       - name: Checkout Debezium UI
         uses: actions/checkout@v2
-      - name: Set up JDK 11
-        uses: actions/setup-java@v1
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
         with:
-          java-version: 11
+          distribution: 'temurin'
+          java-version: 17
```

Comment:
Don't we also want to check out the Debezium core repository like we do in the other connector repositories and make sure we compile the latest source from upstream and store those artifacts before testing Debezium UI?

Reply:
Didn't we agree to explicitly depend on 2.0 instead of latest until we manually update? ` latest` might block too many PRs until breaking changes from the main repo are resolved?

Reply:
@Naros I intend to do this on a separate PR.